config.yaml

```diff
@@ -1,7 +1,9 @@
 options:
-  install_sources: |
-    - http://apt.puppetlabs.com trusty main
-    - http://apt.puppetlabs.com trusty dependencies
-  install_keys: |
+  install_sources:
+    default: |
```

Comment:
kill the pipe (|) at the end here.. it's a list, not a multi-line string that follows.